lhotse/audio.py

```diff
@@ -1264,12 +1265,38 @@ def unmixed_audio(self) -> List[np.ndarray]:
     @property
     def mixed_audio(self) -> np.ndarray:
         """
-        Return a numpy ndarray with the shape (1, num_samples) - a mono mix of the tracks
+        Return a numpy ndarray with the shape (num_channels, num_samples) - a mix of the tracks
+        supplied with ``add_to_mix`` calls.
+        """
+        total = self.num_samples_total
+        mixed = np.zeros((self.num_channels, total), dtype=self.dtype)
+        for offset, track in zip(self.offsets, self.tracks):
+            if track.shape[0] == self.num_channels:
+                # Do nothing
+                ...
+            elif track.shape[0] == 1 and self.num_channels > 1:
+                # Duplicate the track for all channels
+                track = np.tile(track, (self.num_channels, 1))
+            elif track.shape[0] > 1 and self.num_channels == 1:
```

Comment:
is it possible to improve the test coverage of AudioMixer (see codecov comments here)? Historically, it's a class that required many fixes over time.

Reply:
Done.